pytest_html/plugin.py

```diff
@@ -41,11 +41,12 @@ def pytest_addoption(parser):
     group.addoption('--html', action='store', dest='htmlpath',
                     metavar='path', default=None,
                     help='create html report file at given path.')
-    group.addoption('--self-contained-html', action="store",
-                    dest='self_contained_html', default=False,
-                    help='If it is true, asserts that the HTML will not ' +
-                    'depend of external files, risk of breaking CSP. ' +
-                    'DEFAULT: False')
+    group.addoption('--self-contained-html', action='store_true',
+                    help='create a self-contained html file containing all ' +
```

Comment:
The `+` are not needed. See https://docs.python.org/2/reference/lexical_analysis.html#string-literal-concatenation